packages/schematics/.vscode/tasks.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
More or less you are right Matt about the vscode. The issue that I see is that some plugins sometimes try to write there  settings inside the `.vscode` folder. 

I have for example plugin that color every project in different colors. It stores this inside `.vscode` folder so it will trigger a change. Sometimes even small tweaks in `.vscode` will result in some property inside the folder to change. And this is a conflict between developers. 

I'm ok with `tasks.json` and `launch.json` files to exist but inside a documentation file and people could create and modify them as they wish if needed.